doc/build_install/windows/build_osl.py

```diff
@@ -0,0 +1,2355 @@
+#
+# Copyright Contributors to the Open Shading Language project.
+# 
+# Based on USD build_scripts by Pixar Animation Studio
```

Comment:
If this contains any code copied from USD, we need to reproduce their copyright and license information here to be in compliance with that project's Apache 2.0 license.

Reply:
I've added copyright notice to all files
just I'm not sure about the Pixar USD reproduced copyright
please check the file

Reply:
Here's how I think the header of this file should read:

```
# Copyright Contributors to the Open Shading Language project.
# SPDX-License-Identifier: BSD-3-Clause
#
# Portions of this file were derived from USD, which has the following notice:
# Copyright Pixar
# Apache 2.0 license with modifications: https://github.com/PixarAnimationStudios/USD/blob/release/LICENSE.txt
```